src/author.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'item'
+
+class Author
+  attr_accessor :first_name, :last_name
+
+  @items = []
+
```

Comment:
@Khaled-Alaa-1 Your code looks great, but I think `@items = []` should be inside the constructor since it is an instance varible like this
```rb

def initialize(fisrt_name, last_name)
#other variables
@items = []
end
```